grammar.js

```diff
@@ -1619,6 +1619,12 @@ function hl_key_val(left, right) {
   return seq(field("key", left), token.immediate("="), field("val", right));
 }
 
+function cmd_attr_behavior_key_val(left, ...right) {
+  if (right.length > 0)
+    return seq(field("name", token.immediate(left)), token.immediate(","), field("val", ...right));
+  else return field("name", token.immediate(left));
```

Comment:
I don't think that the linter will like that, could you add the brackets here ?

Reply:
It is used in the same way for key_val_arg. I ran prettier this time, I think it only complained because the line was too long. But if you prefer having explicit brackets I can add them